app/javascript/packs/RequestPasswordPage.js

```diff
@@ -0,0 +1,21 @@
+import '@babel/polyfill'
+import 'core-js/es7/object'
+import { isBrowserIE11 } from 'utilities/ie11Utils'
+
+const isIE11 = isBrowserIE11(window)
```

Comment:
We don't support ie11 anymore

Reply:
I know, but I'm not sure if we should remove all IE11 related stuff from everywhere, so I just leave it like the other forms

Reply:
Removing!